apps/masterbots.ai/components/layout/profile/hero.tsx

```
@@ -0,0 +1,42 @@
+  useEffect(() => {
+    if (username) {
+       UserInfoInit()
+    }
+  },[username])
+
+
+  if (!user) return null
+
+    return (
+      <div className="relative bg-left-bottom py-10 bg-[url('/hero-bg.png')] bg-no-repeat ">
+      <div className="absolute inset-0 bg-gradient-to-l from-mirage via-[#2B5D91]/80 to-[#388DE2]/80"></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)